meinberlin/assets/scss/components_user_facing/_follow.scss

```diff
@@ -1,10 +1,16 @@
+.follow__container > span {
+    width: 100%;
+}
```

Comment:
@hom3mad3 this span is the data widget from the template ta hence no class and as it is an inline element it had a limited width (i think anyway) so it was also limiting the nested elements